src/Copy/Text.elm

```diff
@@ -341,3 +342,12 @@ chompTrailingUrlSlash urlString =
 urlToDisplay : String -> String
 urlToDisplay url =
     Url.fromString url |> urlRecombiner |> chompTrailingUrlSlash
+
+
+partnerDescriptionText : Partner -> String
```

Comment:
Ah sorry. I didn't explain it very well. What you need to add is a `Copy.Key` that takes a `String` in the Partner page section - I guess `PartnerDescriptionEmptyText`?

See `PartnerUpcomingEventsText` as an example.

The url display stuff we did is a generic thing for all urls to display, but this is a specific bit of copy.

```

Reply:
ALl of the copy comes from this translation function (`t`) or the APIs.

Reply:
That makes more sense